src/app/(content-layout)/myhistory/_myhistory/components/GroupedByDateTaskItem.tsx

```diff
@@ -1,10 +1,10 @@
 import TaskListItem from '@/components/task-list-item/TaskListItem';
 import { format } from 'date-fns';
-import { TaskDoneProp } from '../types/myhistory-page-type';
+import { GroupedByDateTaskListProps } from '../types/myhistory-page-type';
 
 interface Props {
   date: string;
-  data: TaskDoneProp[];
+  data: GroupedByDateTaskListProps[];
```

Comment:
정확하게 말하면 `GroupedByDateTaskListProps`는 Props가 아니고 Props 안에 있는 데이터 중 하나일 뿐입니다. `GroupedByDateTaskListProps`는 그냥 `GroupedByDateTaskList`가 되는 게 더 좋을 것 같네요. 아래 `GroupedByDateTaskList.tsx`에서도 마찬가지입니다.

Reply:
여기도 key 이름은 그냥 `data`라고 하기보단 명시적으로 적어주는 게 좋아 보입니다.
ex)
```suggestion
groupedByDateTaskList: GroupedByDateTaskList[]
```

Reply:
그리고 궁금한 게 컴포넌트 이름은 `GroupedByDateTaskItem`이라서 개별 아이템을 말하는 것으로 보이는데, 실제로 props에 들어오는 데이터는 `GroupedByDateTaskList`여서 개별 아이템이 아닌 list를 말하는 것으로 보여서 이 부분에서 혼동이 오는 것 같습니다. 특히 `GroupedByDateTaskList`뒤에 `[]`까지 붙어 있어서 리스트의 리스트로 보이는데요, 이런 부분들을 명확히 수정하셔야 좋을 것 같습니다. 코드를 아주 자세히 보기 전까지는 이런 부분을 예측하기가 힘듭니다.

Reply:
https://github.com/FE-team1/coworkers/pull/59#discussion_r2067831341
위에서 추가한 내용처럼 타입 정의에서는 `Item`이라는 글자가 없는 게 더 좋을 것 같습니다. 😅